src/components/taskDetails/TaskDates.tsx

```diff
@@ -71,4 +64,4 @@ const TaskDates: React.FC<TaskDatesProps> = ({
     );
 };
 
-export default React.memo(TaskDates);
+export default TaskDates;
```

Comment:
- remove default export from all components as we're already doing named export

Reply:
done.